CSVSource/README.md

```diff
@@ -0,0 +1,194 @@
+# Overview
+
+This document outlines how to incorporate a CSV Source into your project. The CSV source allows a user to construct applications that detect creation of CSV files in pre-defined folder and upload the file content as events to VANTIQ. The extension source enables control on the names of the different attribute based on the order in the CSV file, and change the name or delete after processing the file. 
+The extension source can handle multiple files parallel;  this is controlled by parameters in the config section . 
+
+The extension source supports only notifications from source to vantiq , those contains the csv converted to json messages based on the schema definition given in the configuration. 
+
+using filewatcher , when new file is created in pre configured folder , support the configurable name file pattern, the context is being build and converted to json . that json is being sent to VANTIQ as incoming event . 
+
+when process start , based on configiration , extension can work on all files exist in the input folder which support the name file pattern , in this way , in case system is failed , it will resnt all accomulated files once it restarted again . 
```

Comment:
```suggestion
When the process starts, based on configuration, extension can work on all files exist in the input folder which support the name file pattern. In this way, in cases where the extension is not running during file updates, it will resend all accumulated files once it restarted. 
```

Reply:
done